src/schema/v2/sales.ts

```diff
@@ -54,13 +54,17 @@ export const SalesConnectionField: GraphQLFieldConfig<void, ResolverContext> = {
   resolve: async (
     _root,
     { ids, isAuction, live, published, sort, registered, ...paginationArgs },
-    { salesLoaderWithHeaders }
+    {
+      unauthenticatedLoaders: { salesLoaderWithHeaders: loaderWithCache },
+      authenticatedLoaders: { salesLoaderWithHeaders: loaderWithoutCache },
+    }
   ) => {
     const { page, size, offset } = convertConnectionArgsToGravityArgs(
       paginationArgs
     )
 
-    const { body: sales, headers } = ((await salesLoaderWithHeaders(
+    const loader = registered ? loaderWithoutCache : loaderWithCache
```

Comment:
#minor if `registered` is a boolean that is passed in (the presence of which indicates the correct loader/cache usage), this might not quite work. Maybe `registered === undefined ? loaderWithoutCache : loaderWithCache` works better?

Reply:
Oh good call, we should probably fix that. We don't have a use case for `registered: false` so it's not extremely urgent, but we'll follow up on this before it becomes too late.